pkg/chartutil/create.go

```diff
@@ -103,6 +103,7 @@ fullnameOverride: ""
 serviceAccount:
   # Specifies whether a service account should be created
   create: true
+  annotations: {}
```

Comment:
I think it would be helpful to add a comment to the `annotations` value

Reply:
True, I was not sure if it was self explanatory enough or not. Thanks, added!